README.md

```diff
@@ -196,6 +196,51 @@ aws_start_me_up = Rouster.new(
 )
 
 ```
+### instantiation for openstack
+
+Rouster allows connection to Openstack instances.
+
+detailed options in ```examples/openstack_example.rb```
+
+```rb
+require 'rouster'
+require 'plugins/openstack' # brings in fog and some helpers
+
+# control a remote opentack machine
+ostack = Rouster.new(
+  :name      => 'ostack-testing',
+  :passthrough => {
+    :type                => :openstack,
+    :openstack_auth_url  => 'http://hostname.domain.com:5000/v2.0/tokens',
+    :openstack_username  => 'some_console_user',
+    :openstack_tenant    => 'tenant_id',
+    :connection_options  => {},
```

Comment:
i wouldn't have an empty hash as an example.. do you have an example of something that could go in here?